### PR TITLE
DomPDF plugin

### PR DESCRIPTION
**Pull request by ozee31**
https://github.com/DaoAndCo/cakephp-dompdf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,6 +268,7 @@ Additional lists you might find useful:
 *Plugins and software for working with PDF files.*
 
 - [CakePdf plugin](https://github.com/FriendsOfCake/CakePdf) - A plugin around PDF generation.
+- [DomPDF](https://github.com/DaoAndCo/cakephp-dompdf) - Generate PDF with DomPDF
```
Comment (dereuromark) [inline README.md]: Full stop missing. `Dompdf` seems to be their wording.

**Comment by dereuromark** [comment]
Tests seem missing here.

**Comment by ADmad** [comment]
What does this plugin offer that `CakePdf` doesn't?

**Comment by ozee31** [comment]
This plugin allows you to easily generate PDF with dompdf. It uses the view's system of CakePHP

**Comment by ADmad** [comment]
> This plugin allows you to easily generate PDF with dompdf. It uses the view's system of CakePHP

So does [CakePdf](https://github.com/FriendsOfCake/cakepdf)

**Comment by ozee31** [comment]
True, it is an alternative

**Comment by ozee31** [comment]
I created helper for help to generate images, css, page break...

**Comment by dereuromark** [comment]
Fair enough.

But you do realize that this is not necessary (and shouldnt be) as the normal helper calls can easily be intercepted so you don't have to modify any code and can continue using the normal helpers?
See http://www.dereuromark.de/2014/04/08/generating-pdfs-with-cakephp/#tips for example.

**Comment by ADmad** [comment]
That helper methods are pretty specific to your needs and not suitable for generic use. They have hardcoded paths and class names.

**Comment by ozee31** [comment]
OK, I'll try to improve my plugin or try your plugin

**Comment by ADmad** [comment]
Closing this PR for now as I don't think it provides anything that CakePdf already doesn't. Since this is meant to be a curated list and not a plugin registry we don't want to list multiple plugins for the same task unless they differ significantly.

**Comment by ozee31** [comment]
OK no problem

**Comment by ADmad** [comment]
You can submit your plugin to be listed on plugins.cakephp.org

**Comment by ozee31** [comment]
ok

**Pull request closed without merging**